### DISCUSSION TITLE
Seemed running 8XA800 but stuck here for 2 hours

### DISCUSSION THREAD

**Discussion by nijisakai** [xai-org/grok-1#210] [General]
Me running it in 8xA800 from China.
![image](https://github.com/xai-org/grok-1/assets/29118280/dc9ae487-2892-4eea-95a5-4157a5e8004f)
and stuck here for like 2 hours yet.
![image](https://github.com/xai-org/grok-1/assets/29118280/4c4df6c8-4c6a-48b1-b9ce-aed3fe43a516)

Should go waiting? Any ideas?

**Comment by nijisakai** [comment]
Although i know the answer is '42'

**Comment by etoyz** [comment]
看起来需要amd的rocm支持，只有Nvidia的CUDA是不行的

**Reply by bluevisor** [reply]
在A800上面跑显示rocm not found是正常的。楼主设置/dev/shm的大小了么？需要320G以上

**Reply by nijisakai** [reply]
> 在A800上面跑显示rocm not found是正常的。楼主设置/dev/shm的大小了么？需要320G以上

谢谢，机器内存500G，也需要设置shm么？
